apps/events/forms.py

```diff
@@ -69,12 +69,12 @@ class ScheduledMessageConfigForm(forms.Form):
         help_text="Instructions for the bot to formulate a response",
         widget=forms.Textarea(attrs={"rows": 5}),
     )
-    frequency = forms.IntegerField(label="Every...", min_value=1)
+    frequency = forms.IntegerField(label="Every...", min_value=0)
```

Comment:
> Does 0 here mean 'not used'?
Correct

> Wouldn't it be better leave the validation as it stands and just ignore the field if it's not being used?
We could do that yes. Ok I'll update

Reply:
See https://github.com/dimagi/open-chat-studio/pull/595/commits/e00b60ba0171bb5dd1a830d95a36081a2a7b2ee9. When we create a one off scheduled message through the UI, the user must specify values for the `frequency` and `repetitions`, since we need the `frequency` to calculate when to ping the user, and the `repetitions` to know when the reminder is over (i.e. 1 triggger > 0 triggers).

In the bot's case, it can give us a date, based on the conversation with the user. In this case, we do not need to have values for `frequency` and `time_period`, since we already know when to ping the user. This is why `non_required_fields` is passed in only from the tools side